Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionsTreeView.cs

```diff
@@ -205,7 +205,7 @@ public override void RedrawUI(ViewState viewState)
             addActionButton.SetEnabled(viewState.actionMapCount > 0);
 
             // Don't want to show action properties if there's no actions.
-            m_Root.Q<VisualElement>("properties-container").visible = m_ActionsTreeView.GetTreeCount() > 0;
+            m_Root.Q<VisualElement>("properties-container").Children().OfType<ScrollView>().First().visible = m_ActionsTreeView.GetTreeCount() > 0;
```

Comment:
Hmm, this is 'ok' if it has the desired result, but I think I would prefer to see the sub area having a name in the UXML file so that you could do something like 

m_Root.Q<VisualElement>("properties-container-without-header").visible = m_ActionsTreeView.GetTreeCount() > 0;

Reply:
I.e. adding here:
https://github.com/Unity-Technologies/InputSystem/blob/dac00ca42d47249b5e711360a326ada6cf6bbed2/Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Resources/InputActionsEditor.uxml#L45

Reply:
Agree, use an id/name to access and check for existence in code.